velox/functions/prestosql/types/fuzzer/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+velox_add_library(velox_presto_types_fuzzer TimestampWithTimeZoneFuzzer.h)
+
+velox_link_libraries(velox_presto_types_fuzzer velox_vector_fuzzer
```

Comment:
Like this it will be linked into the main `libvelox` when building the monolithic version. As fuzzers are always build this is probably not what we want. Use the base functions `add_library` and `target_link_libraries` instead.